bot/code_review_bot/__init__.py

```diff
@@ -57,7 +65,7 @@ def __init__(
         check: str,
         column: int = None,
         message: str = None,
-        level: str = "error",
+        level: Level = Level.Error,
```

Comment:
Maybe warning by default, so when we start publishing all error issues we don't risk publishing some we didn't have to publish

Reply:
Done.